### PR TITLE
OCP 4.15: IPI for PowerVS updates

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.15

Issue: https://issues.redhat.com/browse/MULTIARCH-4115

Link to docs preview: 
- https://70885--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installing-ibm-cloud-account-power-vs#installation-ibm-cloud-iam-policies-api-key_installing-ibm-cloud-account-power-vs
- https://70885--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installing-ibm-power-vs-private-cluster#installation-ibm-power-vs-config-yaml_installing-ibm-power-vs-private-cluster
- https://70885--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_powervs/installing-ibm-cloud-account-power-vs#quotas-and-limits-ibm-power-vs_installing-ibm-cloud-account-power-vs

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -726,9 +727,9 @@ ifdef::ibm-power-vs[]
 
 |platform:
   powervs:
-    cloudConnectionName:
-|The CloudConnectionName is the name of an existing PowerVS Cloud connection.
-|String. For example, `existing_cloudConnectionName`.
+    serviceInstanceGUID:
+|The ServiceInstanceID is the ID of the Power IAAS instance created from the {ibm-cloud-name} Catalog.
```
Comment (mjturek) [inline modules/installation-configuration-parameters.adoc:731]: ServiceInstanceGUID

File: modules/installation-ibm-cloud-regions.adoc [hunk 2]
```diff
@@ -40,29 +40,24 @@ ifdef::ibm-power-vs[]
...
-* `lon` (London, UK)
-** `lon04`
-** `lon06`
-* `osa` (Osaka, Japan)
-** `osa21`
+* `mad` (Madrid, Spain)
+** `mad02`
+** `mad04`
 * `sao` (Sao Paulo, Brazil)
 ** `sao01`
```
Comment (mjturek) [inline modules/installation-ibm-cloud-regions.adoc:50]: we might not get sao01 in before GA. Let's leave it off the docs for now (sorry for not updating you)

**Comment by mjturek** [comment]
Small edits, but looking good!

**Review by mjturek — COMMENTED** [review]
Small edits requested, but looking good!

**Comment by alishaIBM** [comment]
@mjturek Thank you for the comments. I have resolved them. Request you to check. Thanks.

File: modules/quotas-and-limits-ibm-power-vs.adoc [hunk 3]
```diff
@@ -28,14 +28,9 @@ You can create additional `LoadBalancer` service objects to create additional AL
...
-== Cloud connections
+== Transit Gateways
 
-There is a limit of two cloud connections per {ibm-power-server-name} instance. It is recommended that you have only one cloud connection in your {ibm-power-server-name} instance to serve your cluster.
-
-[NOTE]
-====
-Cloud Connections are no longer supported in `dal10`. A transit gateway is used instead.
-====
+Each {product-title} cluster creates its own Transit Gateway to enable communication with a VPC. The default quota of transit gateways per region is 10. If you have 10 transit gateways created, you will need to increase your quota before attempting an installation.
```
Comment (mjturek) [inline modules/quotas-and-limits-ibm-power-vs.adoc:33]:
```
the default quota of transit gateways per account is 10
```

Sorry 1 more small clarification

Comment (alishaIBM) [reply modules/quotas-and-limits-ibm-power-vs.adoc]: Sure @mjturek. Have updated the PR.

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Pull request merged**